terraform/gandi.tf

```diff
@@ -0,0 +1,11 @@
+# FIXME: Not declared because needs the owners block and so would require us
```

Comment:
Would storing it in sops and using the `obfuscated` options work?

Reply:
Actually maybe we just don't bother with this? Does require a lot of info.

Reply:
Yep, i think this would add unnecessary complexity.

Since the domain is already created, you don't need to create it and  you could not delete the domain since this is not allowed by the public API :/